### PR TITLE
feat: impelement costumizable theme and add few basic variable

### PR DESCRIPTION
**Pull request by LordAldi**
fixes (#16)

- implement theme with croco
- change sass to less
- add few basic variables
- change script for start, build, and test 

image

### CODE REVIEW

File: craco.config.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+module.exports = {
+  plugins: [
+    {
+      plugin: CracoLessPlugin,
+      options: {
+        lessLoaderOptions: {
+          lessOptions: {
+            modifyVars: {
+              "@primary-color": "#FDE54E",
```
Comment (ariefrahmansyah) [inline craco.config.js:11]: From where you get these values? Have you confirmed this color theme with UI/UX team?

**Review by ariefrahmansyah — APPROVED** [review]
LGTM from me. We can merge it and update the color theme later (I asked about this on the comment above just for clarity).

File: craco.config.js (hunk 1 above)
Comment (LordAldi) [reply craco.config.js:11]: From ui,ux doc on drive and double check in figma

**Review by fadelarmando — APPROVED** [review]
LGTM

**Pull request merged**